src/main/java/com/exasol/projectkeeper/validators/changesfile/ChangesFileValidator.java

```diff
@@ -99,7 +99,7 @@ private ChangesFile fixSections(final ChangesFile changesFile) {
     private ChangesFile getTemplate() {
         final var changesFile = ChangesFile.builder()
                 .setHeader(List.of("# " + this.projectName + " " + this.projectVersion + ", released "
-                        + LocalDateTime.now().getYear() + "-??-??", ""))//
+                        + LocalDateTime.now().getYear() + "-??-??", "", "Code name:", ""))//
```

Comment:
Will the code name be placed on a new line?

Reply:
yes. Each element in the list is a line.